src/main/java/uk/gov/hmcts/sscs/service/SubmitAppealService.java

```diff
@@ -53,12 +66,21 @@ public void submitAppeal(SyaCaseWrapper appeal) {
         String appellantUniqueId = String.format(ID_FORMAT, appellantLastName, nino.substring(nino.length() - 3));
 
         try {
+            CaseData caseData = submitYourAppealToCcdCaseDataDeserializer.convertSyaToCcdCaseData(appeal);
+            Subscription subscription = caseData.getSubscriptions().getAppellantSubscription().toBuilder()
+                    .tya(appealNumberGenerator.generate())
+                    .build();
+            caseData.getSubscriptions().setAppellantSubscription(subscription);
+            ccdService.createCase(caseData);
             Map<String, Object> placeholders = Collections.singletonMap(SYA_CASE_WRAPPER, appeal);
             byte[] pdf = pdfServiceClient.generateFromHtml(getTemplate(), placeholders);
 
             submitYourAppealEmail.setSubject(appellantUniqueId);
             submitYourAppealEmail.setAttachments(newArrayList(pdf(pdf,appellantUniqueId + ".pdf")));
             emailService.sendEmail(submitYourAppealEmail);
+        } catch (CcdException ccdEx) {
+            LOG.error("Error while creating case in CCD", ccdEx);
```

Comment:
Add AppInsight with relevant log information.

Reply:
will update.

Reply:
This was already added with appinsight log.